packages/app/src/debug/empty/DebugNoRuns.vue

```diff
@@ -1,24 +1,73 @@
 <template>
-  <DebugEmptyView
-    :title="t('debugPage.emptyStates.recordYourFirstRun')"
-    :description="t('debugPage.emptyStates.almostThere')"
-    :example-test-name="t('debugPage.emptyStates.noRunsTestMessage')"
-    :slideshow-campaign="DEBUG_SLIDESHOW.campaigns.recordRun"
-    :help-link-sr-text="t('debugPage.emptyStates.learnAboutRecordingSrText')"
-    help-link-href="https://on.cypress.io/cypress-run-record-key"
-  >
-    <template #cta>
-      <RecordPromptAdapter class="w-[80%]" />
-    </template>
-  </DebugEmptyView>
+  <div class="m-[10%]">
+    <Promo
+      :campaign="DEBUG_PROMO_CAMPAIGNS.recordRun"
+      :medium="DEBUG_TAB_MEDIUM"
+    >
+      <template #header>
+        <PromoHeader :title="t('debugPage.emptyStates.noRuns.title')">
+          <template #description>
+            <i18n-t
+              scope="global"
+              keypath="debugPage.emptyStates.noRuns.description"
+            >
+              <ExternalLink :href="guideUrl">
+                {{ t('debugPage.emptyStates.noRuns.link') }}
+              </ExternalLink>
+            </i18n-t>
+          </template>
+          <template #content>
+            <RecordPromptAdapter />
+          </template>
+        </PromoHeader>
+      </template>
+
+      <template #cards="{ step, goForward, reset }">
+        <TourCard
+          v-if="step === 0"
+          :action="goForward"
+        />
+        <GuideCard1
+          v-else-if="step === 1"
+          :action="goForward"
+        />
+        <GuideCard2
+          v-else-if="step === 2"
+          :action="goForward"
+        />
+        <GuideCard3
+          v-else-if="step === 3"
+          :action="reset"
+        />
+      </template>
+    </Promo>
+  </div>
 </template>
 
 <script lang="ts" setup>
-import DebugEmptyView from './DebugEmptyView.vue'
 import { useI18n } from '@cy/i18n'
+import { computed } from 'vue'
+import { DEBUG_PROMO_CAMPAIGNS, DEBUG_TAB_MEDIUM } from '../utils/constants'
+import Promo from '@packages/frontend-shared/src/gql-components/promo/Promo.vue'
+import PromoHeader from '@packages/frontend-shared/src/gql-components/promo/PromoHeader.vue'
 import RecordPromptAdapter from '@cy/gql-components/RecordPromptAdapter.vue'
-import { DEBUG_SLIDESHOW } from '../utils/constants'
+import GuideCard1 from '../guide/GuideCard1.vue'
```

Comment:
Just a suggestion for future PRs, I think we can do this like you often see in React using a namespace:

```ts
export const Guide = {
  Card1,
  Card2
}

// ...
<template>
  <Guide.Card1 />
  <Guide.Card2 />
</template>
```

Reply:
This is fine too, though 👍

Reply:
Love it, definitely will restructure. I knew exactly how I would structure all of this state & these components in React but was struggling to map that into the "Vue way"

Reply:
Sure, no problem - not sure why this is less common in Vue world, same patterns should generally work.

Reply:
LMK when you do this (or if you decide not to) and I can approve, everything else is looking pretty good to me.

Reply:
I ended up writing a simple wrapper for each of the sets of cards (`DebugGuide`, `DebugTour`, etc) so the card components don't have to be referenced en-masse outside that directory. Given how the different slideshows were structured I didn't end up seeing much organizational benefit from the namespacing suggestion